modules/UI/shared_video/SharedVideo.js

```diff
@@ -252,10 +250,10 @@ export default class SharedVideoManager {
             // in onPlayerStateChange
             player.playVideo();
 
-            let thumb = new SharedVideoThumb(
-                self.url, SHARED_VIDEO_CONTAINER_TYPE, VideoLayout);
-            thumb.setDisplayName(player.getVideoData().title);
-            VideoLayout.addRemoteVideoContainer(self.url, thumb);
+            VideoLayout.addSharedVideoThumbnail(
+                self.url,
+                player.getVideoData().title,
+                SHARED_VIDEO_CONTAINER_TYPE);
```

Comment:
The question becomes about how this would work in redux/react. SharedVideo would not be creating a react element and passing it somewhere for it to be appended. It should be the responsibility of a parent component to create child components, which to me means filmstrip creating thumbnails. How exactly to go about that is still open for discussion because it has not been implemented. What I'm seeing with this change is removing the responsibility of thumbnail creation more towards filmstrip. If you do disagree with that flow then now is a good time to speak up before more changes are made.